.gitlab-ci.yml

```diff
@@ -1,216 +1,32 @@
-.check_skip_rust:
-  before_script:
-    - |-
-      if echo "$CI_COMMIT_TITLE" | grep -q "ci-skip-rust"; then
-        echo "Skipping rust tests due to commit title"
-        exit 0
-      fi
-
-.check_skip_integration_tests:
-  before_script:
-    - |-
-      if echo "$CI_COMMIT_TITLE" | grep -q "ci-skip-integration-tests"; then
-        echo "Skipping integration tests due to commit title"
-        exit 0
-      fi
+include:
+  - local: '.gitlab/check.yml'
+  - local: '.gitlab/test.yml'
+  - local: '.gitlab/build.yml'
+  - local: '.gitlab/release.yml'
+
+variables:
+  CARGO_HOME: $CI_PROJECT_DIR/.cargo
+  CARGO_TARGET_DIR: $CI_PROJECT_DIR/target
+  DOCKER_HUB_PARACHAIN: "kiltprotocol/kilt-node"
+  DOCKER_HUB_STANDALONE: "kiltprotocol/standalone-node"
+  DOCKER_HUB_DIP_PROVIDER_TEMPLATE: "kiltprotocol/dip-provider-node-template"
+  DOCKER_HUB_DIP_CONSUMER_TEMPLATE: "kiltprotocol/dip-consumer-node-template"
+  TRY_RUNTIME_CLI_VERSION_TAG: v0.7.0
```

Comment:
Why are these variables declared here? Can't we declare them where they are needed?